README.md

```diff
@@ -1,14 +1,9 @@
-# assessor-portal-packer 💀📦 #
+# assessor-portal-packer 🕳📦 #
 
 [![GitHub Build Status](https://github.com/cisagov/assessor-portal-packer/workflows/build/badge.svg)](https://github.com/cisagov/assessor-portal-packer/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started. This skeleton project
-contains [licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+This repository is used to create images containing the Assessor Portal
+software.
 
 ## Pre-requisites ##
 
```

Comment:
Please update the `Using Your Local Environment` section to include the `BUILD_BUCKET` environment variable as a requirement for the Packer template.

Reply:
Good catch!  Rectified in https://github.com/cisagov/assessor-portal-packer/pull/1/commits/3fad09095574c0b01545002b0eeb24dd69640382.